clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -550,9 +570,9 @@ compute_irregular_trace_windows(std::vector<instr_interval_t> &instr_intervals)
     // must have a duration long enough to cover the end of the program.
     irregular_window_ptr =
         (irregular_window_t *)dr_global_alloc(sizeof(irregular_window_t));
-    // DELAY_FOREVER_THRESHOLD might be too small for long traces, but it doesn't matter
-    // because we trace_for_instrs = 0, so no window is created anyway.
-    irregular_window_ptr->no_trace_for_instrs = DELAY_FOREVER_THRESHOLD;
+    // A no_trace_for_instrs window of 0 means: delay forever, don't start a tracing
```

Comment:
What if the user wants to have an "infinite trace" instead of "infinite no trace" at the end?

Reply:
Then they need to make the last instruction interval large enough to cover the end of the program.